ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/os_service.py

```diff
@@ -154,9 +210,9 @@
 RE_UPSTART_SERVICE = re.compile('(?P<title>[\w\-]+(\s\([\w\/]+\))?)\s'   # service title
                                 '(?P<active_status>[\w\/]+)'             # active status
                                 '(.\sprocess\s(?P<main_pid>\d+))?')      # active status if exists
-RE_SYSTEMV_SERVICE = re.compile('(?P<title>[A-Za-z0-9_\-\.\s:]+)'
-                                '((\s|:)\(pid\s+(?P<main_pid>\d+)\))?'
-                                '\sis\s(?P<active_status>[\w\s]+)')
+# Only links that start with 'S' are running in current runlevel
+RE_SYSTEMV_SERVICE = re.compile(ur'S[0-9]+[a-zA-Z0-9\-_]+\s->\s\.\./init\.d/'
+                                '(?P<title>.+)')
```

Comment:
[a-zA-Z0-9\-_] can be reduced to [\w\-], 
Also explain briefly what pattern you are trying to catch.